README.md

```diff
@@ -3,7 +3,7 @@ Framework to automate development of Algorand Assets and Smart Contracts.
 
 ## Objectives
 
-Algorand builder is an trustworthy framework for Algorand dapps (Decentralized Applications). Main goal is to make shipping Algorand applications simple, efficient, and scalable. Think about it as a truffle suite for Algorand. The framework provides following functionality:
+Algorand builder is an trustworthy framework for Algorand dapps (Decentralized Applications). Its main goal is to make shipping Algorand applications simple, efficient, and scalable. Think about it as a Truffle suite for Algorand. The framework provides following functionality:
```

Comment:
Its -> It's

Reply:
Its means that I point into the previous thing (Polish: tego).

`https://www.thefreedictionary.com/its`:

belonging to it: This is its handle.
Not to be confused with:
it's – it is: It’s cold outside.; it has: It’s been a long winter.